_docs/services/aws-elasticsearch.md

```diff
@@ -20,11 +20,11 @@ This service is currently running Elasticsearch 7.4.  Our prior Elasticsearch se
 
 ## Plans
 
-Service Name | Plan Name | Description |
------------- | --------- | ----------- | -----
-`aws-elasticsearch` | `es-dev` | Single data node for non-prod use only |
-`aws-elasticsearch` | `es-medium` | 3 Master and 2 Data node cluster |
-`aws-elasticsearch` | `es-medium-ha` | 3 Master and 4 Data node cluster |
+Service Name | Plan Name | Description | Num of nodes |
```

Comment:
Same as above - change "num" to "number."

Reply:
adding now to see how it looks